apps/web/src/components/evaluations/ConfigurationForm.tsx

```diff
@@ -299,25 +323,22 @@ export function ConfigurationAdvancedForm<
           <ActualOutputTest configuration={testConfiguration} />
         </div>
       )}
-      {doesntRequireExpectedOutput && setIssueId && (
+      {doesntRequireExpectedOutput && (
         <FormFieldGroup
           label='Link issue'
           description='Link the evaluation to an issue in your agent logs'
           layout='horizontal'
         >
           <Select
             searchable={true}
-            // TODO(eval-generation): This will not get the issueId from page 2,3, etc. need to change with Andres feedback
-            value={issueId ?? null}
+            removable={true}
```

Comment:
No need for `removable={true}` when is `true`. Just `removable`

<img width="480" height="332" alt="Image" src="https://github.com/user-attachments/assets/cccf39a0-116d-4e0f-ae61-8c9157c8ac37" />